tests/ui-toml/item_name_repetitions/threshold0/clippy.toml

```diff
@@ -0,0 +1 @@
+struct-field-name-threshold = 0
```

Comment:
```suggestion
struct-field-name-threshold = 0
enum-variant-name-threshold = 0
```
It seems you forgot to add the config option for enums

Reply:
Also, while we're at it, can the `item_names.rs` file in the threshold 5 test also be renamed `item_name_repetitions.rs` for consistency?

Reply:
should be fixed now :)